paasta_tools/api/views/instance.py

```diff
@@ -207,6 +214,22 @@ def kubernetes_instance_status(
             verbose=verbose,
             pod_list=pod_list,
         )
+
+        if not omit_smartstack:
```

Comment:
Instead of the double-negative, can you just make this a bool for `include_smartstack` ?

Reply:
OK, change to `include_smartstack`. This probably breaks paasta status temporary because the swagger change.